web/src/containers/Summary/components/DigitalAssets.js

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import { Select } from 'antd';
+
+import withConfig from 'components/ConfigProvider/withConfig';
+import Markets from './Markets';
+import { MarketsSelector } from 'containers/Trade/utils';
+import { EditWrapper, IconTitle } from 'components';
+import STRINGS from 'config/localizedStrings';
+
+const DigitalAssets = (props) => {
+	const {
+		user,
+		coins,
+		balance,
+		oraclePrices,
+		pairs,
+		activeTheme,
+		router,
+		pair,
+		markets,
+		icons: ICONS,
+	} = props;
+	const [options, setOptions] = useState([
+		{ value: STRINGS['ALL'], label: STRINGS['ALL'] },
+	]);
+	const [selectedSource, setSelectedSource] = useState('');
+
+	useEffect(() => {
+		getSearchResult();
+		handleOptions();
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
+
+	const handleBack = () => {
+		router.goBack();
+	};
+
+	const handleOptions = () => {
+		const optionData = [];
+		const nonDublicateCoins = [];
+		markets.forEach((market) => {
+			const pair_2_symbol = market?.pairTwo?.symbol;
+			if (!nonDublicateCoins.includes(pair_2_symbol)) {
+				nonDublicateCoins.push(pair_2_symbol);
+				optionData.push({
+					value: pair_2_symbol,
+					label: pair_2_symbol,
+				});
+			}
+		});
+		setOptions([...options, ...optionData]);
+	};
+
+	const getSearchResult = () => {
+		const result = {};
+		Object.entries(coins).map(([key, obj]) => {
+			const hasCoinBalance = !!balance[`${key}_balance`];
+			if (hasCoinBalance) {
+				result[key] = { ...obj, oraclePrice: oraclePrices[key] };
+			}
+			return key;
+		});
+		return { ...result };
+	};
+
+	return (
+		<div className="digital-market-wrapper">
+			<div className="market-wrapper">
+				<div className="header-container">
+					<div className="d-flex">
+						<IconTitle
+							className="digital-assets-icon"
+							stringId="DIGITAL_ASSETS.DIGITAL_ASSETS_TITLE"
+							text={STRINGS['DIGITAL_ASSETS.DIGITAL_ASSETS_TITLE']}
+							iconPath={ICONS['ASSET_INFO_COIN']}
+							iconId="ASSET_INFO_COIN"
+							textType="title"
+						/>
+					</div>
+					<div className="link-content" onClick={handleBack}>
+						<EditWrapper stringId="DIGITAL_ASSETS.GO_BACK">
+							&lt; {STRINGS['DIGITAL_ASSETS.GO_BACK']}
+						</EditWrapper>
+					</div>
+				</div>
+				<div className="d-flex justify-content-between mb-3">
+					<div>
+						<div className="gray-text">
```

Comment:
I think the already defined `secondary-text` class can be used instead of the gray-text class all over the component.

Reply:
Updated the `secondary-text` class instead of  `gray-text`